crates/uv/src/commands/workspace/list.rs

```diff
@@ -0,0 +1,36 @@
+use std::fmt::Write;
+use std::path::Path;
+
+use anyhow::Result;
+
+use owo_colors::OwoColorize;
+use uv_preview::{Preview, PreviewFeatures};
+use uv_warnings::warn_user;
+use uv_workspace::{DiscoveryOptions, Workspace, WorkspaceCache};
+
+use crate::commands::ExitStatus;
+use crate::printer::Printer;
+
+/// List workspace members
+pub(crate) async fn list(
+    project_dir: &Path,
+    preview: Preview,
+    printer: Printer,
+) -> Result<ExitStatus> {
+    if preview.is_enabled(PreviewFeatures::WORKSPACE_LIST) {
```

Comment:
Looks like this is supposed to be:

```suggestion
    if !preview.is_enabled(PreviewFeatures::WORKSPACE_LIST) {
```